skbuild/resources/cmake/UseF2PY.cmake

```diff
@@ -0,0 +1,135 @@
+#.rst:
+#
+# The following functions are defined:
+#
+# .. cmake:command:: add_f2py_target
+#
+# Create a custom rule to generate the source code for a Python extension module
+# using f2py.
+#
+#   add_f2py_target(<Name> [<F2PYInput>]
+#                   [OUTPUT_VAR <OutputVar>])
+#
+# ``<Name>`` is the name of the new target, and ``<F2PYInput>``
+# is the path to a pyf source file.  Note that, despite the name, no new
```

Comment:
is it the convention to name all fortran files with `.pyf` extension ? Or is it okay to have some with extension like `.f90` ?

Reply:
F2PY can be run over *.f files, but for building extensions, we usually run it over these *.pyf files. Here's an example: https://github.com/scipy/scipy/blob/e6a4d07dac39e1c257fccdac666bf28e024f70bd/scipy/integrate/dop.pyf

Reply:
See the build log [here](https://ci.appveyor.com/project/scipy/scipy/build/1.0.1832/job/xlyww6t8ipnnc176). You can Ctrl+F for the F2PY invocations.